app/services/notifications/new_comment/send.rb

```diff
@@ -35,6 +35,7 @@ def call
         end
 
         # Send PNs using Rpush - respecting users' notificaton delivery settings
+        # TODO: [@msarit] update query to call mobile_comment_notifications from correct table
```

Comment:
I need help reworking this query; `mobile_comment_notifications` now lives in `users_notification_setting`, not `users`.

Reply:
This is a variation on the same pattern:

```ruby
User
  .joins(:notification_setting)
  .where(notification_setting: { mobile_comment_notifications: true })
  .ids
```